util/file.go

```diff
@@ -627,3 +628,25 @@ func IsDirectoryEmpty(dirPath string) (bool, error) {
 	}
 	return true, nil
 }
+
+// FindFileInParentDirs returns `path` if the file exists, otherwise searches for a file with the same name in the parent directories and returns the file's path if found.
+func FindFileInParentDirs(path string) string {
```

Comment:
Sorry, should've mentioned it: https://github.com/gruntwork-io/terragrunt/blob/master/config/config_helpers.go#L410

Reply:
Oh, no, it's my fault. I should have known about this feature. Thanks for the advice! Redone.